index.html

```diff
@@ -53,123 +62,126 @@ <h1 class="heading-primary2">I’m a software developer</h1>
-                <li class="flex-items">JavaScript</li>
-                <li class="flex-items">html</li>
-              </lu>
+          <div class="outer-grid">
+            <div class="picture"></div>
+            <div class="outer-container">
+              <div>
+                <h2 class="multi-post">Multi-Post Stories Gain+Glory</h2>
+              </div>
+              <div class="flex-container-div">
+                <ul class="flex-container">
+                  <li class="flex-items"> Ruby on rails</li>
+                  <li class="flex-items">css</li>
+                  <li class="flex-items">JavaScript</li>
+                  <li class="flex-items">html</li>
+                </ul>
+              </div>
+              <div><a href="#" class="btn-see-project">See project</a></div>
             </div>
-            <div><a href="#" class="btn-see-project">See project</a></div>
           </div>
-        </div>
-        <div class="outer-grid">
-          <div class="picture"></div>
-          <div class="outer-container">
-            <div>
-              <h2 class="multi-post">Multi-Post Stories Gain+Glory</h2>
+          <div class="outer-grid">
+            <div class="picture"></div>
+            <div class="outer-container">
+              <div>
+                <h2 class="multi-post">Multi-Post Stories Gain+Glory</h2>
+              </div>
+              <div class="flex-container-div">
+                <ul class="flex-container">
+                  <li class="flex-items"> Ruby on rails</li>
+                  <li class="flex-items">css</li>
+                  <li class="flex-items">JavaScript</li>
+                  <li class="flex-items">html</li>
+                </ul>
+              </div>
+              <div><a href="#" class="btn-see-project">See project</a></div>
             </div>
-            <div >
-              <lu class="flex-container">
-                <li class="flex-items"> Ruby on rails</li>
-                <li class="flex-items">css</li>
-                <li class="flex-items">JavaScript</li>
-                <li class="flex-items">html</li>
-              </lu>
-            </div>
-            <div><a href="#" class="btn-see-project">See project</a></div>
           </div>
         </div>
       </section>
       <section class="about-myself">
         <div class="about-me">
           <h1 class="heading-about-me">About me</h1>
           <div class="supporting about">
-            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-              don’t hestiate to contact me.</p>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
           </div>
           <div><a href="#" class="my-resume">Get My Resume</a></div>
```

Comment:
- [ ] Kindly fix this button here. :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200886436-4cc42e3d-a965-40a9-9bab-4f5217a192b0.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200886549-4b035796-f346-4a99-92dc-9ef4bc3dfc6f.png)